lib/krane/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module Krane
-  VERSION = "3.7.0"
+  VERSION = "3.7.1"
```

Comment:
I'd argue this could be 3.8.0 since there's an additive change to the public CLI API, but I don't feel _that_ strongly